plugins/aerospike/aerospike.go

```diff
@@ -0,0 +1,338 @@
+package aerospike
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+	"sync"
+)
+
+import (
+	"github.com/influxdb/telegraf/plugins"
```

Comment:
Just the way I do it, anything that needs a go get is in the second import statement. Looking around it seems like that clashes with the current style, so I will change it to conform with that.